src/universal_transfer_operator/data_providers/constants.py

```diff
@@ -0,0 +1,71 @@
+from enum import Enum
+
+
+class SupportedDataProvider(Enum):
+    """
+    List of all the supported DataProviders.
+    """
+
+    LOCAL = "local"
+    S3 = "s3"
+    FTP = "ftp"
+    HDFS = "hdfs"
+    SFTP = "sftp"
+    GCS = "gcs"
+    MYSQL = "mysql"
+    POSTGRES = "postgres"
+    BIGQUERY = "bigquery"
+    REDSHIFT = "redshift"
+
+
+class Database(Enum):
+    """
+    List of all the supported Databases DataProviders
+    """
+
+    POSTGRES = "postgres"
+    POSTGRESQL = "postgres"
+    SQLITE = "sqlite"
+    BIGQUERY = "bigquery"
+    SNOWFLAKE = "snowflake"
+
+
+class FileSystem(Enum):
```

Comment:
What are your thoughts on naming this `FileLocation`, as we did in Astro?
The reason is that S3 and GCS are not file systems. They are object stores.